locations/spiders/department_veterans_affairs.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import datetime
+from locations.items import GeojsonPointItem
+
+
+class TemplateSpider(scrapy.Spider):
+    name = "department_veterans_affairs"
+    allowed_domains = ["api.va.gov"]
+
+    def start_requests(self):
+
+        for i in range(1, 117):
+            URL = 'https://api.va.gov/v0/facilities/va?address=United%%20States&bbox[]=-180&bbox[]=90&bbox[]=180&bbox[]=-90&type=all&page=%s' % i
+            yield scrapy.Request(URL, callback=self.parse_info)
+
+
+    def store_hours(self, store_hours):
+        if not store_hours:
```

Comment:
You might be able to simplify the opening hours code by using the `OpeningHours` class. See [here](https://github.com/alltheplaces/alltheplaces/blob/f508a2cec9a1d24300bbabaad511abd369fe35e3/locations/spiders/chase.py#L17) for an example of how to use it.